config.go

```diff
@@ -35,12 +35,16 @@ type Config struct {
 	EventsPort int
 	// the interface we should be listening on for events
 	EventsInterface string
+	// dcos token
+	MarathonToken string
 	// HTTPBasicAuthUser is the http basic auth
 	HTTPBasicAuthUser string
 	// HTTPBasicPassword is the http basic password
 	HTTPBasicPassword string
 	// CallbackURL custom callback url
 	CallbackURL string
+	// Token for DCOS environment
+	MarathonToken string
```

Comment:
I'm not sure about the name. Is the token specific to DCOS's "flavor" of Marathon or something that vanilla Marathon supports too? Can you possibly point me at some piece of documentation where the token is described?